server/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "npm i && node server.js"
+    "start": "node ./app/routes/index.js"
```

Comment:
i dont understand why make an `app.js` file and then use a different file to run the server ?

Reply:
the app.js is not used to run the server, its used to deal with static files and so on

Reply:
oh my bad then

Reply:
@aysam199 but why are we using handlebars ? instead of making it work with React ?

Reply:
I'm a bit confused, the handlebars and express are for the backend, i make a request from the front end with react to be handled by the backend handlebars. I cant make it just work with react i need it on the backend aswell

Reply:
yes you need the backend, for the data. but the react is the one that handles the frontend.
handlebars also does the same as react, so we can't have 2 frontend frameworks, we need only 1